change/@microsoft-teams-js-393fc2b1-d5c9-413a-98d5-b44bf8a88903.json

```diff
@@ -1,6 +1,6 @@
 {
-  "type": "none",
-  "comment": "Added missing documentation to the Pages capability",
+  "type": "patch",
```

Comment:
For my own edification: why is this a patch? I would have assumed it was none because it had no effect on the shipping package we're delivering. Because it affects documentation?

Reply:
Yes, I could see going either way, but I think since it's not obvious to me, it's better to have it in the changelog than not. I'm open to other perspectives!

Reply:
My only perspective is that we should make sure we have a consistent rule about what all of the different types mean so we make sure the CHANGELOG gets updated deterministically.